### PR TITLE
✨ Code-Review cleanup

### PR DESCRIPTION
**Pull request by laurentsimon**
* **Please check if the PR fulfills these requirements**
- [X] Tests for the changes have been added (for bug fixes / features)
- [X] PR title follows the guidelines defined in  https://github.com/ossf/scorecard/blob/main/CONTRIBUTING.md#pr-process

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
fix

* **What is the current behavior?** (You can also link to an open issue here)
see https://github.com/ossf/scorecard/issues/737

* **What is the new behavior (if this is a feature change)?**
see https://github.com/ossf/scorecard/issues/737

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Other information**:

### CODE REVIEW

File: checker/check_result.go [hunk 1]
```diff
@@ -85,6 +85,45 @@ type CheckResult struct {
 	Reason   string        `json:"-"` // A sentence describing the check result (score, etc)
 }
```
Comment (laurentsimon) [inline checker/check_result.go:87]: ignore this file. Will disappear from the PR once https://github.com/ossf/scorecard/pull/739 is merged.

File: checker/check_result.go [hunk 2]
```diff
@@ -85,6 +85,45 @@ type CheckResult struct {
...
 
+// CreateProportionalScore() creates a proportional score.
+func CreateProportionalScore(b, t int) int {
```
Comment (laurentsimon) [inline checker/check_result.go:89]: ignore this file except the `AggregateScoresWithWeight` function. Will disappear from the PR once https://github.com/ossf/scorecard/pull/739 is merged.

**Review by azeemshaikh38 — APPROVED** [review]

**Comment by laurentsimon** [comment]
@naveensrinivasan @inferno-chromium anyone can explain to me what the failing check means?

**Comment by inferno-chromium** [comment]
> @naveensrinivasan @inferno-chromium anyone can explain to me what the failing check means?

is checks.md correct ?, the failure is some text on auto-generated checks.md not matching.

```
 M checks/checks.md
diff --git a/checks/checks.md b/checks/checks.md
index 6ff0554..93bbb0a 100644
--- a/checks/checks.md
+++ b/checks/checks.md
....
```

**Comment by azeemshaikh38** [comment]
> > @naveensrinivasan @inferno-chromium anyone can explain to me what the failing check means?
> 
> is checks.md correct ?, the failure is some text on auto-generated checks.md not matching.
> 
> ```
>  M checks/checks.md
> diff --git a/checks/checks.md b/checks/checks.md
> index 6ff0554..93bbb0a 100644
> --- a/checks/checks.md
> +++ b/checks/checks.md
> ....
> ```

Maybe in a previous PR we modified `checks.md` directly instead of `checks.yaml`?

**Comment by laurentsimon** [comment]
> > @naveensrinivasan @inferno-chromium anyone can explain to me what the failing check means?
> 
> is checks.md correct ?, the failure is some text on auto-generated checks.md not matching.
> 
> ```
>  M checks/checks.md
> diff --git a/checks/checks.md b/checks/checks.md
> index 6ff0554..93bbb0a 100644
> --- a/checks/checks.md
> +++ b/checks/checks.md
> ....
> ```

this PR does not modify the checks.md, so I don't understand why the error :/

**Comment by inferno-chromium** [comment]
> > > @naveensrinivasan @inferno-chromium anyone can explain to me what the failing check means?
> > 
> > 
> > is checks.md correct ?, the failure is some text on auto-generated checks.md not matching.
> > ```
> >  M checks/checks.md
> > diff --git a/checks/checks.md b/checks/checks.md
> > index 6ff0554..93bbb0a 100644
> > --- a/checks/checks.md
> > +++ b/checks/checks.md
> > ....
> > ```
> 
> this PR does not modify the checks.md, so I don't understand why the error :/

previous cl seems to cause the difference, see
https://github.com/ossf/scorecard/blob/main/checks/checks.md#security-policy
and this https://github.com/ossf/scorecard/blob/main/checks/checks.yaml#L339
the content in yaml and md don't match, so just need to fix it in this cl for tests to pass.

**Pull request merged**